shallow_backup/utils.py

```diff
@@ -78,6 +78,12 @@ def mkdir_warn_overwrite(path):
 		print_path_blue("Created directory:", path)
 
 
+def empty_backup_dir_check(backup_path):
+	if not os.path.isdir(backup_path) or not os.listdir(backup_path):
+		print_red_bold('No package backups found.')
```

Comment:
This message will depend on the method that calls it. Maybe pass the keyword as a second param?

Reply:
Looks good to me.